.github/good-first-issue.md

```diff
@@ -0,0 +1,13 @@
+The issue was marked with the `good first issue` label by a maintainer.
+
+This means that it is a good candidate for someone interested in contributing to the project, but does not know where to start.
+
+Have a look at the [Contributing Guide](https://github.com/vercel/next.js/blob/canary/contributing.md) first.
+
+This will help you set up your development environment to get started. When you are ready, open a PR, and link back to this issue in the form of adding "Fixes #1234" to the PR description, where "1234" is the issue number. This will auto-close the issue when the PR gets merged, making it easier for us to keep track of what has been fixed.
```

Comment:
```suggestion
To get started, read the [Contributing Guide](https://github.com/vercel/next.js/blob/canary/contributing.md). When you are ready, open a PR and link back to this issue in the form of adding `Fixes #1234` to the PR description, where `1234` is the issue number. This will automatically close the issue when the PR gets merged, making it easier for us to keep track of what has been fixed.
```